cs-amazon/src/main/java/io/cloudslang/content/amazon/actions/lambda/InvokeLambdaAction.java

```diff
@@ -0,0 +1,125 @@
+     * @param payload           Optional - Lambda function payload in JSON format
+     *                          Example: "{"key1":"value1", "key1":"value2"}"
+     *                          Default: null
+     * @return                  A map with strings as keys and strings as values that contains: outcome of the action, returnCode of the
+     *                          operation, or failure message and the exception if there is one
+     */
+    @Action(name = "Invoke Lambda Function",
+            outputs = {
+                    @Output(Outputs.RETURN_CODE),
+                    @Output(Outputs.RETURN_RESULT),
+                    @Output(Outputs.EXCEPTION)
+            },
+            responses = {
+                    @Response(text = Outputs.SUCCESS, field = Outputs.RETURN_CODE, value = Outputs.SUCCESS_RETURN_CODE,
+                            matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.RESOLVED),
+                    @Response(text = Outputs.FAILURE, field = Outputs.RETURN_CODE, value = Outputs.FAILURE_RETURN_CODE,
+                            matchType = MatchType.COMPARE_EQUAL, responseType = ResponseType.ERROR)
+            }
+    )
+    public Map<String, String> execute(
+                                            @Param(value = IDENTITY,   required = true)  String identity,
+                                            @Param(value = CREDENTIAL, required = true, encrypted = true)  String credential,
+                                            @Param(value = REGION,     required = true)  String region,
+                                            @Param(value = PROXY_HOST)                   String proxyHost,
+                                            @Param(value = PROXY_PORT)                   int    proxyPort,
+                                            @Param(value = PROXY_USERNAME)               String proxyUsername,
+                                            @Param(value = PROXY_PASSWORD)               String proxyPassword,
+                                            @Param(value = FUNCTION_NAME, required = true) String function,
+                                            @Param(value = FUNCTION_QUALIFIER)          String qualifier,
+                                            @Param(value = FUNCTION_PAYLOAD)            String payload,
+                                            @Param(value = CONNECT_TIMEOUT)             int connectTimeoutMs,
+                                            @Param(value = EXECUTION_TIMEOUT)           int execTimeoutMs) {
+
+            ClientConfiguration clientConf = new ClientConfiguration()
+                    .withConnectionTimeout(connectTimeoutMs)
+                    .withProxyHost(proxyHost)
+                    .withProxyPort(proxyPort)
+                    .withProxyUsername(proxyUsername)
+                    .withProxyPassword(proxyPassword);
+
+            AWSLambdaAsyncClient client = (AWSLambdaAsyncClient) AWSLambdaAsyncClientBuilder.standard()
```

Comment:
the cast here is not necessarily required (the client would work the same as a `AWSLambdaAsync` instance), but if you want to keep it for code clarity, it can be left like that